### PR TITLE
📊  Adding total fertility rate from gapminder

### PR DESCRIPTION
**Pull request by spoonerf**

### CODE REVIEW

File: snapshots/gapminder/2023-09-22/total_fertility_rate.xlsx.dvc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  origin:
+    # Data product / Snapshot
+    title: Babies per woman (total fertility rate)
+    description: |-
+      Sources
+      — 1800 to 1950 (and in some cases also years after 1950): Gapminder v6 which were compiled and documented by Mattias Lindgren, see previous versions further down on this page.
+
+      — 1950 to 2014: In most cases we use the latest UN estimates from World Population Prospects 2017 published in the file with Annually interpolated demographic indicators, called WPP2017_INT_F01_ANNUAL_DEMOGRAPHIC_INDICATORS.xlsx , accessed on September 2, 2017.
+
+      — 2015 – 2099: We use the UN forecast of future fertility rate in all countries, called median fertility variant.
```
Comment (lucasrodes) [inline snapshots/gapminder/2023-09-22/total_fertility_rate.xlsx.dvc:13]: is UN WPP 2022 not suitable for this?

File: snapshots/gapminder/2023-09-22/total_fertility_rate.xlsx.dvc [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/en/latest/architecture/metadata/reference/origin/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Babies per woman (total fertility rate)
+    description: |-
+      Sources
```
Comment (lucasrodes) [inline snapshots/gapminder/2023-09-22/total_fertility_rate.xlsx.dvc:8]: maybe you can try bolding `**Sources**`

File: etl/steps/data/garden/gapminder/2023-09-22/total_fertility_rate.meta.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+      children_dying_before_five_per_woman:
+        title: Children that died before five years age per woman
+        unit: under-five deaths per woman
+        display:
+          numDecimalPlaces: 1
+      children_surviving_past_five_per_woman:
+        title: Children that survived past five years age per woman
+        unit: children surviving past five years per woman
+        display:
+          numDecimalPlaces: 1
```
Comment (lucasrodes) [inline etl/steps/data/garden/gapminder/2023-09-22/total_fertility_rate.meta.yml:16]: would be nice to add some description fields. Maybe `description_short`.

**Review by lucasrodes — APPROVED** [review]
Minor suggestions

File: snapshots/gapminder/2023-09-22/total_fertility_rate.xlsx.dvc (hunk 1 above)
Comment (spoonerf) [reply snapshots/gapminder/2023-09-22/total_fertility_rate.xlsx.dvc:13]: Hmm good point, it probably would be best to use UN WPP from 1950 onwards.

Comment (spoonerf) [reply snapshots/gapminder/2023-09-22/total_fertility_rate.xlsx.dvc:14]: I've added this in now, the metadata doesn't seem to include UN WPP though. Is currently better to just manually curate the metadata for cases like this?

**Pull request merged**